src/components/AoRadio.vue

```diff
@@ -1,36 +1,43 @@
 <template>
   <div class="ao-radio">
-    <label class="ao-radio__label">
-      <input
-        v-model="checked"
-        v-bind="$attrs"
-        :value="val"
-        :disabled="disabled"
-        type="radio"
-        class="ao-radio__input"
-        @change="toggle"
-        @blur="emitBlur"
-        @focus="emitFocus"
-      >
-      <div
-        v-show="showLabel"
-        class="ao-radio__content"
-      >
-        <span class="ao-radio__label-text">
-          {{ label }}
-        </span>
-        <p
-          v-if="infoText"
-          class="ao-radio__info-text"
-        >
-          {{ infoText }}
-        </p>
-      </div>
+    <input
+      :id="uniqId"
+      v-model="checked"
+      v-bind="$attrs"
+      :value="val"
+      :disabled="disabled"
+      type="radio"
+      class="ao-radio__input"
+      @change="toggle"
+      @blur="emitBlur"
+      @focus="emitFocus"
+    >
+    <label
+      :for="uniqId"
+      class="ao-radio__label-text-default"
+    >
+      {{ label }}
     </label>
+    <div
+      v-show="showLabel"
```

Comment:
this will duplicate the label for view using `showLabel: true`.

Reply:
I kept this because new added label won't break original UI even `showLabel: true`, the added label just for meet accessibility requirement.

Reply:
oh, trick css `font-size: 0;` here,

Reply:
did you test with a view using `showLabel: false`? just want to make sure it didn't break the view